.github/actions/setup-postgres/.env

```
@@ -0,0 +1,5 @@
+POSTGRES_USER=postgres
+POSTGRES_OPTIONS="-c max_connections=1000 -c shared_buffers=2GB"
+POSTGRES_PASSWORD_=postgres
```

```suggestion
POSTGRES_PASSWORD=postgres
```
Was that trailing underscore supposed to be there?

---

[139df8e](https://github.com/smartcontractkit/chainlink/pull/7771/commits/139df8ee5e43c19f71af03d3117a5f621f44ac44)